### PR TITLE
Update Vite, Parcel and Webpack guides

### PR DESCRIPTION
**Pull request by julien-deramond**
### Description

Based on the recent modifications that happened in the `examples` repo, I've updated Vite, Parcel and Webpack guides to match the new configurations.

/cc @XhmikosR 

#### Parcel

* Dropped the tildes

#### Vite

* Added a `build` part to the `vite.config.js` file
* Moved the `<script>` tag in the `index.html` file into the `<head>`
* Dropped the first step of "Import Bootstrap" section as it's not needed anymore
* Rewrote the second step for consistency with Parcel guide for instance
* Dropped the tildes

#### Webpack

* Added the new `html-webpack-plugin` dependency and a sentence to explain it
* Changed the commands to create the project folder and files
* Changed the `webpack.config.js` file to match the new one in `examples` repository
* Don't use `dist/index.html` anymore but `src/index.html` instead
* Regenerated the diffs linked to `miniCssExtractPlugin`

### Motivation & Context

Consistency across the docs and the examples repo.

### Type of changes

- [x] Refactoring (non-breaking change)

### Checklist

- [x] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)
- [x] My code follows the code style of the project _(using `npm run lint`)_
- [x] My change introduces changes to the documentation
- [x] I have updated the documentation accordingly
- (N/A) I have added tests to cover my changes
- [x] All new and existing tests passed

#### Live previews

- <https://deploy-preview-38305--twbs-bootstrap.netlify.app/docs/5.3/getting-started/parcel/>
- <https://deploy-preview-38305--twbs-bootstrap.netlify.app/docs/5.3/getting-started/vite/>
- <https://deploy-preview-38305--twbs-bootstrap.netlify.app/docs/5.3/getting-started/webpack/>

### CODE REVIEW

File: site/content/docs/5.3/getting-started/webpack.md [hunk 1]
```diff
@@ -148,7 +153,10 @@ Importing Bootstrap into Webpack requires the loaders we installed in the first
 1. **Set up the loaders in `webpack.config.js`.** Your configuration file is now complete and should match the snippet below. The only new part here is the `module` section.
 
    ```js
+   'use strict'
+
    const path = require('path')
+   const HtmlWebpackPlugin = require('html-webpack-plugin')
```
Comment (XhmikosR) [inline site/content/docs/5.3/getting-started/webpack.md:159]: we need autoprefixer here too and maybe in the install instruction if it's not mentioned

Comment (julien-deramond) [reply site/content/docs/5.3/getting-started/webpack.md:159]: Good catch you're right. Let me fix it

Comment (XhmikosR) [reply site/content/docs/5.3/getting-started/webpack.md:159]: And probably postcss and cli too.

Comment (XhmikosR) [reply site/content/docs/5.3/getting-started/webpack.md:160]: I know they are not in it but won't the file error out if the devDependencies are not installed?

Haven't read through these guides, just thinking out loud here.

File: site/content/docs/5.3/getting-started/webpack.md [hunk 2]
```diff
@@ -130,7 +130,7 @@ With dependencies installed and our project folder ready for us to start coding,
      "scripts": {
        "start": "webpack serve",
        "build": "webpack build",
-       "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (julien-deramond) [inline site/content/docs/5.3/getting-started/webpack.md:128]: Please revert this change or modify the three guides the same way by also adding information about it in the paragraph. Because otherwise, something is missing when you follow the tutorial.

Comment (XhmikosR) [reply site/content/docs/5.3/getting-started/webpack.md:128]: Nothing is missing. There's a build script which is just called if one runs test.

Comment (XhmikosR) [reply site/content/docs/5.3/getting-started/webpack.md:128]: But anyway, let's keep the moot default for now

**Review by XhmikosR — APPROVED** [review]

File: site/content/docs/5.3/getting-started/webpack.md (hunk 2 above)
Comment (julien-deramond) [reply site/content/docs/5.3/getting-started/webpack.md]: > Nothing is missing. There's a build script which is just called if one runs test.

Again, the problem was not the modification itself. But in the previous paragraph, something's missing because you didn't explain: "we're gonna change the `test` command to use a basic one to at least build the project blah blah blah". And if we do it, we must do it in the 3 guides.
Let's modify it later on then.
